packages/core/src/routes/account-center/index.ts

```diff
@@ -29,18 +34,23 @@ export default function accountCentersRoutes<T extends ManagementApiRouter>(
       body: z.object({
         enabled: z.boolean().optional(),
         fields: accountCenterFieldControlGuard.optional(),
+        webauthnRelatedOrigins: webauthnRelatedOriginsGuard.optional(),
       }),
       response: AccountCenters.guard,
       status: [200],
     }),
 
     async (ctx, next) => {
-      const { enabled, fields } = ctx.guard.body;
+      const { enabled, fields, webauthnRelatedOrigins } = ctx.guard.body;
+
       // Make sure the account center exists
       await findDefaultAccountCenter();
       const updatedAccountCenter = await updateDefaultAccountCenter({
         enabled,
         fields,
+        webauthnRelatedOrigins: webauthnRelatedOrigins
+          ? deduplicate(webauthnRelatedOrigins)
+          : undefined,
```

Comment:
If webauthnRelatedOrigins is undefined here, will it remove all existing webauthnRelatedOrigins stored in the config? Is that the expected behavior?